.changeset/disable-reactions-broadcast.md

```diff
@@ -0,0 +1,6 @@
+---
+"@rocket.chat/meteor": patch
```

Comment:
```suggestion
"@rocket.chat/meteor": minor
```

new feats are labeled as minor releases on the changeset, it goes like
fix: patch
feat: minor
feat! or fix! (breaking changes): major